api/api/accounts/tests.py

```diff
@@ -0,0 +1,132 @@
+import pytest
+from django.contrib.auth import get_user_model
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+User = get_user_model()
+userCreationEndpoint = '/users/'
```

Comment:
variables should follow snake_case: https://peps.python.org/pep-0008/#function-and-variable-names